### PR TITLE
Update solidity.md to create an Aragon App instead

### PR DESCRIPTION
**Pull request by sohkai**
Discussed a few months ago, but this updates our solidity job opening to a new, more generic challenge given we now maintain a payroll app.

### CODE REVIEW

File: docs/jobs/openings/solidity.md [hunk 1]
```diff
@@ -22,36 +22,6 @@ We are looking for an Ethereum wizard to work with us on [Aragon Core](https://g
...
-
-  /* EMPLOYEE ONLY */
-  function determineAllocation(address[] tokens, uint256[] distribution); // only callable once every 6 months
-  function payday(); // only callable once a month
-
-  /* ORACLE ONLY */
-  function setExchangeRate(address token, uint256 usdExchangeRate); // uses decimals from token
-}
-```
+- Write the contract component of a new Aragon App that either improves upon one of our current [apps](https://github.com/aragon/aragon-apps) or is an entirely new use case that you think will wow us. The final contract should be deployable to the EVM and published to GitHub (public or private). Send us the link to the repo plus any information you consider important to *hiring@aragon.one*, with the subject `Solidity opening application`.
```
Comment (Smokyish) [inline docs/jobs/openings/solidity.md]: > contract component of a new Aragon App that either improves upon one of our current [apps]

I feel this is a bit vague, a `new app that improves on a current app` could be opened more to give some details of what's expected, is it a new app or improvement to the existing codebase.

> deployable to the EVM
Does any EVM instance work? In that case there would ie. be no gas limitations to consider?

File: docs/jobs/openings/solidity.md [hunk 2]
```diff
@@ -22,36 +22,6 @@ We are looking for an Ethereum wizard to work with us on [Aragon Core](https://g
...
-
-  /* EMPLOYEE ONLY */
-  function determineAllocation(address[] tokens, uint256[] distribution); // only callable once every 6 months
-  function payday(); // only callable once a month
-
-  /* ORACLE ONLY */
-  function setExchangeRate(address token, uint256 usdExchangeRate); // uses decimals from token
-}
-```
+Write the contract component of a new Aragon App that either improves upon one of our current [apps](https://github.com/aragon/aragon-apps) or is an entirely new use case that you think will wow us. The final contract should be deployable to the EVM and published to GitHub (public or private). Send us the link to the repo plus any information you consider important to *hiring@aragon.one*, with the subject `Solidity opening application`.
```
Comment (Smokyish) [inline docs/jobs/openings/solidity.md]: > contract component of a new Aragon App that either improves upon one of our current [apps]

I feel this is a bit vague, a `new app that improves on a current app` could be opened more to give some details of what's expected, is it a new app or improvement to the existing codebase.

> deployable to the EVM
Does any EVM instance work? In that case there would ie. be no gas limitations to consider?

Comment (sohkai) [reply docs/jobs/openings/solidity.md]: I wanted to keep it vague to let "improvements" be in any number of ways they felt was worthwhile, no matter if that was a big feature, huge technical improvement, or etc.

Ultimately, my thinking with this is that it'll be up to them to prove why they thought their app is valuable.

Would it help if I added a sentence to clarify "improvements," e.g. `You get to choose what to do, no matter if that's an entirely new use case, huge additional feature, awesome optimization, etc.`?

----------

> Does any EVM instance work? In that case there would ie. be no gas limitations to consider?

I'd say so, I'll clarify the constraints.

Comment (Smokyish) [reply docs/jobs/openings/solidity.md]: Cool, thanks @sohkai !

**Pull request merged**